src/utils.ts

```diff
@@ -77,3 +70,25 @@ export const isPeerVideoJid = function (jid: string) {
 export const isVideoJid = function (jid: string): boolean {
   return jid && !!jid.match(/@conference/) && !isAcdJid(jid);
 };
+
+export const logPendingSession = function (
+  logger: ILogger,
+  message: string,
+  pendingSession: IPendingSession | ISessionInfo,
+  level: LogLevels = 'info'
+): void {
+  const data: any = {
+    sessionId: (pendingSession as IPendingSession).id || (pendingSession as ISessionInfo).sessionId,
+    autoAnswer: pendingSession.autoAnswer,
+    conversationId: pendingSession.conversationId,
+    originalRoomJid: pendingSession.originalRoomJid,
+    fromUserId: pendingSession.fromUserId
```

Comment:
we probably need to leave out the `originalRoomJid`

Reply:
Done